workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -512,6 +514,7 @@ def _get_credentials(self, storage):
 
 class NFSMounter(StorageMounter):
     available = False
+    sensitive_policy = PipelineAPI(os.environ['API'], None).get_preference(SENSITIVE_POLICY_PREFERENCE)['value']
```

Comment:
I think it is better to pass policy in the constructor args instead of initialising it for each mounter